cases/tcp_to_tcp_performance/configurations/big-vms/terraform.tfvars

```diff
@@ -0,0 +1,11 @@
+# WARNING!
+#
+# Do not modify the parameters of this file since historical test results are
+# based on these parameters. Please create a new configuration and specify taht
+# configuration instead.
+producer_instance_type  = "c5.large"
+producer_instance_count = 5
+subject_instance_type    = "c5.2xlarge"
```

Comment:
:( I'm just realizing this isn't nearly large enough. If you look at https://github.com/timberio/vector/issues/391, you'll notice we see a steep decline at 16 cores. Given everything that has changed since then, we'll want to test at least 16 cores or test against 4, 8, and 16 to see if we decline.